lib/migrations/__tests__/migration.js

```diff
@@ -40,7 +59,6 @@ describe('migrations', function () {
         return new _migration2.default(configuration);
       }).not.to.throwError();
     });
-
     describe('countHits', function () {
```

Comment:
No tests for count required as count api remains the same: https://github.com/elastic/elasticsearch-js-legacy/blob/128ba27f1ab6609034f7400df4f98e059c2b27c9/docs/api_methods_7_4.asciidoc#count